docs/user_guide.md

```diff
@@ -402,3 +402,37 @@ you can see the item named "flink-pod-monitor" in the "Service Discovery" sectio
 ### Manage savepoints
 
 See this [doc](./savepoints_guide.md) on how to manage savepoints with the operator.
+
+### Flink Job update
+
+To update a running Flink job's program or execution settings, you can create new savepoint, terminate the job,
+and create a new FlinkCluster with the new program, settings, and savepoint. 
+However, in some cases, you may want to update the Flink job while maintaining the logical continuity
+of the Flink job with the FlinkCluster custom resource. In this case, you can continuously update
+the FlinkCluster custom resource, and the Flink operator takes care of the process required to update
+the Flink job and cluster.
+
+There are several points to note when using the job update feature.
+* To use the job update feature, `savepointsDir` must be set and the value of this field cannot be deleted when updating.
+This is because the Flink operator requires the value to create a savepoint for job updates.
+* You can replay Flink job from your desired savepoint by updating `fromSavepoint`.
+If you want to resume the updated job from the latest savepoint, set `fromSavepoint` to `LATEST`.
```

Comment:
What is the expected behavior if there is no savepoint taken yet.

Reply:
If the job is stopped and a savepoint cannot be created, taking savepint will be aborted and the job will be started without `--fromSavepoint`. This is also true when creating a new FlinkClsuter.

If the job is running and the savepoint creation fails, the operator will keep trying until the savepoint creation succeeds. In this case, there is a possibility that a savepoint will be created, so the user will have to check the status of the Flink cluster.

Reply:
In the case of updating, It seems to be improved if jobStatus.fromSavepoint is avaiable.

Reply:
Fixed convertFromSavepoint to resume the job from jobStatus.fromSavepoint, when there is no latest savepoint and jobStatus.fromSavepoint is available.

Reply:
We need to add this information in the user guide, so that users can better understand the expected behavior.